rules/aip0146/any_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0146
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestAny(t *testing.T) {
+	for _, test := range []struct {
+		name     string
+		Package  string
+		Type     string
+		problems testutils.Problems
+	}{
+		{"Valid", "foo", "string", nil},
+		{"IgnoredCommon", "google.longrunning", "google.protobuf.Any", nil},
+		{"InvalidAny", "foo", "google.protobuf.Any", testutils.Problems{{Message: "Any"}}},
```

Comment:
Will the message really just be `"Any"`? I thought it would be `"Avoid using google.protobuf.Any fields in public APIs."`?

Reply:
The way `testutils.Problems` works is it uses `strings.Contains` logic. This avoids having tests that are too precise and veer toward change detection.